lib/matplotlib/projections/polar.py

```diff
@@ -1173,6 +1173,11 @@ def set_thetagrids(self, angles, labels=None, frac=None, fmt=None,
 
         ACCEPTS: sequence of floats
         """
+        if frac is not None:
+            cbook.warn_deprecated('2.1', name='frac', obj_type='parameter',
```

Comment:
Can this be `2.2` (or maybe `2.1.1`), since `2.1` has already been released?

Reply:
I put `2.1` because it technically stopped working in that version. Not sure how much effort is worth it to re-enable the functionality.